### DISCUSSION TITLE
Layout Image incorrect on location of priority

### DISCUSSION THREAD

**Discussion by jecxjo** [todotxt/todo.txt#53] [Q&A]
Completed Task, Rule 2 states

> The date of completion appears directly after the x, separated by a space.

In the layout image in the format rules section, the priority is shown between the `x` and the completion date when the rule states otherwise. Should be

`x 2016-05-20 (A) 2016-04-30 measure space for +chapelShelving @chapel due:2016-05-30`

**Comment by davidstosik** [comment]
Wow that's a good point! I wonder what's the actual intention... 🤔 

In SimpleTask on Android, if I choose to keep priority on completed tasks, I get `x 2019-07-03 (C) 2019-07-03 Test completed`.

todotxt_cli removes the priority: `x 2019-07-03 2019-07-03 test` (I'd prefer keeping it though...)

**Comment by TangentFoxy** [comment]
Interesting to note that [Todo.txt for Android](https://play.google.com/store/apps/details?id=net.c306.ttsuper) handles this by replacing the priority label with a key:value tag (i.e. `pri:B`). I'm working on my own project related to this, I'll make it able to handle either format for import and export with the key:value solution.
